### PR TITLE
Create a toggle to represent and change state of trigger from TriggesActivity #124 :step-1

### PR DESCRIPTION
**Pull request by shubhamvashisht**
i have commented out some code which will be needed in next step.

### CODE REVIEW

File: mobile/src/main/java/org/hawkular/client/android/backend/BackendPipes.java [hunk 1]
```diff
@@ -72,6 +72,7 @@ private Paths() {
         public static final String OPERATIONS = "inventory/traversal/f;{f}/rt;{rt}/type=ot";
         public static final String PERSONA = "hawkular/metrics/tenants";
         public static final String TRIGGERS = "alerts/triggers";
+        public static final String TRIGGER_STATUS = "triggers/enabled";
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/backend/BackendPipes.java]: this should be part of next PR. Please revert this from this PR

File: mobile/src/main/java/org/hawkular/client/android/adapter/TriggersAdapter.java [hunk 2]
```diff
@@ -92,12 +93,13 @@ public void onClick(View view) {
             }
         });
 
-        viewHolder.menuButton.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View view) {
-                triggerListener.onTriggerMenuClick(view, position);
+        viewHolder.toggleTrigger.setChecked(trigger.getEnableStatus());
+
+        /*viewHolder.toggleTrigger.setOnCheckedChangeListener(new CompoundButton.OnCheckedChangeListener() {
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/adapter/TriggersAdapter.java]: If this commented code is no longer needed. Remove that

Comment (KeenWarrior) [reply mobile/src/main/java/org/hawkular/client/android/adapter/TriggersAdapter.java]: I will suggest to use `onTriggerToggleChanged` only at place of `setChecked`

File: mobile/src/main/java/org/hawkular/client/android/backend/model/Trigger.java [hunk 3]
```diff
@@ -74,13 +82,15 @@ public Trigger createFromParcel(Parcel parcel) {
     private Trigger(Parcel parcel) {
         this.id = parcel.readString();
         this.description = parcel.readString();
+        this.enabled = (boolean) parcel.readValue(null);
```
Comment (KeenWarrior) [inline mobile/src/main/java/org/hawkular/client/android/backend/model/Trigger.java]: As from json `true` and `false will be received as Strings. You can have a look [example](https://github.com/hawkular/hawkular-android-client/blob/master/mobile/src/main/java/org/hawkular/client/android/backend/model/MetricAvailabilityBucket.java#L42) for how you should save and receive them

**Review by KeenWarrior — CHANGES_REQUESTED** [review]

**Comment by shubhamvashisht** [comment]
@garg-anuj changes done sir.

**Review by KeenWarrior — APPROVED** [review]
Looking good to me

**Pull request merged**